src/components/StarSlider/StarSlider.js

```diff
@@ -11,12 +11,27 @@ class StarSlider extends Component {
     }
   }
 
+  submitRating = event => {
```

Comment:
We might want to consider pulling all of our fetch calls into an apiCalls component today.  When we do that, it might require some minor changes in how components like this access those functions.